app/views/application/_flash.html.erb

```diff
@@ -1,6 +1,9 @@
 <% flash.each do |type, message| %>
-  <div data-alert class="alert-box <%= type %> radius">
+  <% type = 'success' if type == 'notice' %>
```

Comment:
Devise adds the class 'notice' which isn't part of Foundation. Converts that to 'success' instead.